tests/hats/io/test_validation.py

```diff
@@ -155,7 +153,7 @@ def test_valid_catalog_strict_all(small_sky_source_dir, small_sky_order1_dir, sm
     flags = {
         "strict": True,  # more intensive checks
         "fail_fast": False,  # check everything, and just return true/false
-        "verbose": False,  # don't bother printing anything.
+        "verbose": True,  # don't bother printing anything.
```

Comment:
The comment is at odds with the behavior now, it seems; which is desired?

Reply:
Keen eyes! I was debugging something and wanted more verbosity. Reverted the change.